terraform-provider-constellation/examples/full/aws_cluster.tf

```diff
@@ -70,6 +70,7 @@ module "aws_infrastructure" {
   image_id                                = data.constellation_image.bar.image.reference
   region                                  = local.region
   zone                                    = local.zone
+  internal_load_balancer                  = false
   debug                                   = false
```

Comment:
These values are false by default and thus could be removed, right?

Reply:
Yes but I like to use this as a reference on what this module exposes (also since internal LB is quite a used and relevant feature).

Reply:
Fair enough. I think we could also make a point for keeping the examples as minimal as possible for conciseness, but no strong opinion here

Reply:
I like it so that one essentially doen't even need the docs if one remembers what the options do but don't always remember the spelling.